Django/Meetify/appapi/chat.py

```diff
@@ -0,0 +1,40 @@
+import json
+from django.http import HttpResponse
+from django.http import JsonResponse
+
+from ..models import Matches, Messages
+from ..serializers import MessagesSerializer
+
+
+def get_messages(request, user_id):
+    if user_id != request.user.pk:
+        return HttpResponse(status=401, reason="User must be logged in to view messages")
+        
+    messages = Messages.objects.filter(MatchId__in=(Matches.objects.filter(User1_id=user_id) | Matches.objects.filter(User2_id=user_id)))
```

Comment:
`|` is a bitwise operator in Python, right? Just want to verify that's what you meant, as opposed to something like `or`

Reply:
The `|` operator in Django queries like this represents an `OR` SQL condition. So yes, the use of it here is intentional.

Reply:
Ah interesting, I forgot you can override operators like that in custom classes. No worries then!